Meta/CMake/jakt.cmake

```diff
@@ -31,6 +27,55 @@ else()
 endif()
 set(JAKT_LIBRARY_DIR "${JAKT_LIBRARY_BASE}/${JAKT_TARGET_TRIPLE}")
 set(JAKT_INCLUDE_DIR "${CMAKE_SYSROOT}/usr/local/include/runtime")
+
+# Make sure the jakt compiler:
+# - exists
+# - can do C++ interop
+# Otherwise, complain and just disable jakt for the project.
+if (ENABLE_JAKT)
+    if (NOT EXISTS ${JAKT_COMPILER})
+        message(WARNING "Jakt compiler not found at ${JAKT_COMPILER}, disabling jakt")
+        set(ENABLE_JAKT OFF CACHE BOOL "Enable jakt" FORCE)
+    else()
+        file(WRITE ${CMAKE_BINARY_DIR}/jakt_test.jakt "import extern \"LibMain/Main.h\"\nfn main() { }\n")
```

Comment:
there's probably some try_compile() "best practices" that could be used here to define a `try_compile_jakt` function, but this should do the trick for now.

Reply:
Sure, let's seeeeee

Reply:
Ehhhh that generates a cmakelists and runs it, we don't need that (we're not even compiling to binary, just making sure jakt itself works)